docs/integration/alerts-pagerduty-integration.md

```diff
@@ -35,9 +35,17 @@ Navigate to [Notification Hooks](https://apps.sematext.com/ui/hooks/create) (in
 
 ![Sematext Notification Hooks](https://sematext.com/docs/images/integrations/sematext-notification-hooks.png  "Sematext Notification Hook")
 
-Enter required parameters and copy incoming PagerDuty Service API Key. Click Test button to confirm that Sematext app is sending data and save your PagerDuty alerts integration.
+Enter required parameters and copy incoming PagerDuty Key. Click Test button to confirm that Sematext app is sending data and save your PagerDuty alerts integration.
```

Comment:
Enter required => Enter the required
copy incoming => copy the incoming
Click Test => click the Test
app => App... although.... it's not really Sematext *App* that is sending the data, so maybe just "... that Sematext is ..."